src/main/scala/com/lucidworks/spark/SolrRelation.scala

```diff
@@ -41,18 +43,27 @@ class SolrRelation(
   def this(parameters: Map[String, String], sqlContext: SQLContext) {
     this(parameters, sqlContext, None)
   }
-
+  var collection=conf.getCollection.get
   checkRequiredParams()
   // Warn about unknown parameters
   val unknownParams = SolrRelation.checkUnknownParams(parameters.keySet)
   if (unknownParams.nonEmpty)
     log.warn("Unknown parameters passed to query: " + unknownParams.toString())
 
   val sc = sqlContext.sparkContext
+
+ if(conf.timeSeriesPartitionOn.getOrElse(false)) {
+   val feature=new PartitionByTimeFeature(conf)
+   val p=new PartitionByTimeAssignmentStrategy(feature,conf)
+   val allCollections=p.getPartitionsForQuery()
+   collection=allCollections mkString ","
+   }
```

Comment:
this block is poorly formatted and does not match the style of the surrounding code